events/api.py

```diff
@@ -3592,10 +3592,29 @@ def filter_queryset(self, queryset):
             if show_all:
                 # displays all editable events, including drafts, and public non-editable events
                 queryset = editable_queryset | public_queryset
+
             admin_user = self.request.query_params.get("admin_user")
+            registration_admin_user = self.request.query_params.get(
+                "registration_admin_user"
+            )
+            if admin_user and registration_admin_user:
+                raise ParseError(
+                    "Supply either 'admin_user' or 'registration_admin_user', not both"
+                )
+
             if admin_user:
                 # displays all editable events, including drafts, but no other public events
                 queryset = editable_queryset
+
+            if registration_admin_user:
```

Comment:
It would be nice to move this filtering logic from here into `EventFilter` class

Reply:
Moved the filterting filtering login to `EventFilter` in commit https://github.com/City-of-Helsinki/linkedevents/pull/768/commits/df538b49e32d0d766c61b41da1db4b3957962e68

Reply:
Great! Would be nice if the changes would be introduced in the commit  [`Event endpoint registration_admin_user parameter`](https://github.com/City-of-Helsinki/linkedevents/pull/768/commits/cfeed00255c580185f5b21de7da3f8ff0169dfbc) => so some rebasing to avoid the churn.

Reply:
Cleaned the commit history little bit and squashed  fix commits to most suitable commits